src/trackers/TVC.py

```
@@ -389,108 +481,186 @@ async def search_existing(self, meta, disctype):
+                desc += f"[b]This season premiered on:[/b] {channel} on {airdate}\n"
+                if meta.get('episodes'):
+                    desc += "\n\n[b]Episode List[/b]\n\n"
+                    for ep in meta['episodes']:
+                        ep_num = ep.get('code', '')
+                        ep_title = ep.get('title', '').strip()
+                        ep_date = ep.get('airdate', '')
+                        ep_overview = ep.get('overview', '').strip()
+
+                    desc += f"[b]{ep_num}[/b]"
+                    if ep_title:
+                        desc += f" – {ep_title}"
```

Ruff (0.14.4)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)